spokestack/extensions/webrtc/agc.pyx

```diff
@@ -0,0 +1,94 @@
+"""
+This module contains the AutomaticGainControl class implemented in Cython
+"""
+# distutils: sources = filter_audio/agc/analog_agc.c, filter_audio/agc/digital_agc.c
+# distutils: include_dirs = filter_audio/agc/include/
+cimport numpy as np
+cimport cagc
+
+
+MIC_MAX = 255
+MIC_TARGET = 180
+
+np.import_array()
```

Comment:
Is this statement necessary? I remember removing it in a version I was compiling and everything was fine. If it is needed, disregard this comment.